inc/namespace.php

```diff
@@ -108,8 +108,12 @@ function remove_2fa_dummy_provider( array $providers ) : array {
  * @return bool
  */
 function override_two_factor_universally_forced( bool $is_forced ) : bool {
+	if ( maybe_skip_two_factor_forcing_for_local_env() ) {
+		return false;
```

Comment:
I think this should return `$is_forced` so it can be configured via admin if desired.

Reply:
Do you mean we want to expose this as an admin setting ?

Without an admin setting, return `$is_forced` will disregard the intention of the change, if `$is_force` is true, it'll bypass the check for a local env and force 2fa which is the opposite of what we're trying to achieve.

Reply:
Unless set via the admin `$is_forced` will default to `false`. So what we're saying is "ignore the Altis config on local but pass through setting in the database".

There's already a network admin setting as part of the work Mike S did, for roles too. The option of using the Altis config bypasses any user setting from the admin and hardcodes it, so that's the part we're skipping on local.